src/bootstrap/doc.rs

```diff
@@ -570,7 +570,7 @@ fn doc_std(
     if builder.no_std(target) == Some(true) {
         panic!(
             "building std documentation for no_std target {target} is not supported\n\
-             Set `docs = false` in the config to disable documentation, or pass `--exclude doc::library`."
+             Set `docs = false` in the config to disable documentation, or pass `--skip library`."
```

Comment:
This changed more that exclude/skip replace?

Reply:
@klensy 
Sorry, but I don't get it. Would you explain more, please?

Reply:
before: `--exclude doc::library`,
after: `--skip library`

Reply:
Probably. Is this suggestion still valid after change?

Reply:
Running this on master:

```bash
x --stage=0 doc --exclude library
```

Output:
```
...truncated
Skipping Set({doc::library}) because it is excluded
...truncated
```


running it on my change:

```bash
x --stage=0 doc --exclude library
```


```
...truncated
Skipping Set({doc::library}) because it is excluded
...truncated
```

Reply:
Hm, ok.